src/main/java/dan200/computercraft/core/filesystem/FileSystem.java

```diff
@@ -697,11 +729,7 @@ public void flush() throws IOException
                     throw new UnsupportedOperationException();
                 }
             };
-            synchronized( m_openFiles )
-            {
-                m_openFiles.add( file );
-            }
-            return file;
+            return (IMountedFileNormal) openFile( file, reader );
```

Comment:
I can, but I wanted a one-liner, which is why `openFile` returns the handle in the first place. So unless that cast is bad for some reason, it's down to preference, I guess.